release-notes/release-notes5106.md

```diff
@@ -0,0 +1,101 @@
+
+`https://example.com/!exclamation-point`
+
+resulted in the following, incomplete, link and trailing text
+
+```html
+<a href="https://example.com/">https://example.com/</a>!exclamation-point
+```
+Equivalent results occurred when URL paths contained the colon character (for example, `https://example.com/:colon`).
+
+With this update, the link detection logic has been improved to capture and present such URLs correctly.
+
+Now, pasting the string 
+
+`https://example.com/!exclamation-point`
+
+results in the following, complete, link and no trailing text
+
+```html
+<a href="https://example.com/!exclamation-point">https://example.com/!exclamation-point</a>
+```
+
+> **NOTE**: Exclamation points and colons are still boundary characters for URL detection and delimiting. If the string `https://example.com/colon:` is pasted or typed into a {{site.productname}} document, automatic link detection will produce a link that does not include the trailing colon character.
+
+For information on the PowerPaste plugin, [PowerPaste plugin]({{site.baseurl}}plugins/premium/powerpaste/)..
+
+
+## Bug fixes
+
+{{site.productname}} 5.10.6 provides fixes for the following bugs:
+
+### The name and id attributes of some elements were incorrectly removed during serialization
+
+Previously, when {{site.productname}} serialized an object, a bug in the SaxParser incorrectly removed `name` and `id` attributes from some elements (for example, `img` and `iframe` elements, among others).
+
+This occurred in several circumstances but could be demonstrated by adding, for example, an image with `id` and `name` attributes to a {{site.productname}} instance and then choosing **Tools > Source code** from the menu bar.
+
+The displayed source code did not include the `id` and `name` attributes.
+
+With this update, {{site.productname}} includes a change in the filtering that prevents the attribute stripping. Consequently, serialization no longer strips attributes from elements.
+
+> **NOTE**: The {{site.productname}} 5.10.6 SaxParser method now works the same way as it does in {{site.productname}} 6.
+
+### Notifications did not properly reposition when toggling fullscreen mode
+
+A {{site.productname}} toolbar option, *Fullscreen*, toggles the {{site.productname}} editor between the size it takes by default and, as the toolbar’s name suggests, fullscreen. Fullscreen is the largest size available to the editor (ie, taking up all the space available to {{site.productname}} in the browser).
+
+If a {{site.productname}} instance had a custom notification, `+editor.notificationManager.open()+`, set to display when the *Fullscreen* button was used, toggling from Fullscreen back to default resulted in the notification displaying outside the {{site.productname}} editor.
+
+{{site.productname}} 5.10.6 corrects this. When toggling between the two *Fullscreen* states, custom notifications now display immediately below the toolbar in either state, as expected.
+
+
+## Changes
```

Comment:
> it’s expected it should fire since the editor resizes

Which argues for the language used in the _Toggling fullscreen mode with the `fullscreen` plugin now also fires the `ResizeEditor` event_ note to be changed.

It currently reads:

> As part of that fix, a change was made to the `fullscreen` plugin.

It should, perhaps, and given ‘expected’ above, read:

> As part of that fix, an error in the `fullscreen` plugin was corrected.

Also, this change constitutes something of an argument for retaining the NOTE above, albeit also with an edit to emphasise the correction:

> **NOTE**: {{site.productname}} 5.10.6 includes a fix to the `fullscreen` plugin. When toggling fullscreen mode with the `fullscreen` plugin it now fires the `ResizeEditor` event, as expected.